shimmingtoolbox/cli/create_coil_profiles.py

```diff
@@ -9,24 +9,31 @@
 import numpy as np
 import pathlib
 import tempfile
+import re
 
-from shimmingtoolbox.coils.create_coil_profiles import create_coil_profiles
+from shimmingtoolbox.coils.create_coil_profiles import create_coil_profiles, get_wire_pattern
+from shimmingtoolbox.coils.biot_savart import generate_coil_bfield
 from shimmingtoolbox.cli.prepare_fieldmap import prepare_fieldmap_uncli
 from shimmingtoolbox.utils import create_output_dir, save_nii_json, set_all_loggers
 from shimmingtoolbox.masking.threshold import threshold as mask_threshold
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+GAMMA = 42.576E6 # in Hz/Tesla
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+@click.group(context_settings=CONTEXT_SETTINGS,
+             help="Create coil profiles according to the specified algorithm as an argument e.g. st_create_coil_profiles xxxxx")
+def coil_profiles_cli():
+    pass
 
```

Comment:
```suggestion


```
Make sure there are 2 blank lines between functions, except in classes where it's only one.